smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolUnitTestRequestGenerator.kt

```diff
@@ -75,20 +76,26 @@ open class HttpProtocolUnitTestRequestGenerator protected constructor(builder: B
     }
 
     private fun renderClientBlock(test: HttpRequestTestCase) {
-        val serviceShape = ctx.service
         val clientName = "${ctx.settings.sdkId}Client"
-
-        if (!serviceShape.getTrait(EndpointRuleSetTrait::class.java).isPresent) {
-            val host: String? = test.host.orElse(null)
-            val url = "http://${host ?: "example.com"}"
-            writer.write("\nlet config = try await $clientName.${clientName}Configuration(endpointResolver: StaticEndpointResolver(endpoint: try \$N(urlString: \$S)))", SmithyHTTPAPITypes.Endpoint, url)
-        } else {
-            writer.write("\nlet config = try await $clientName.${clientName}Configuration()")
+        val region = "us-west-2"
+
+        writer.openBlock("let config = try await \$L.Config(", ")", clientName) {
+            writer.write("awsCredentialIdentityResolver: try \$N(),", SmithyTestUtilTypes.dummyIdentityResolver)
```

Comment:
The dummy AWS identity resolver is passed into the config, so no need to set dummy credentials anywhere else for protocol tests.